src/components/component-playground.js

```diff
@@ -156,26 +157,42 @@ class ComponentPlayground extends Component {
     };
   }
 
+  static getDerivedStateFromProps(nextProps, preState) {
```

Comment:
Another small nit: I think `prevState` is more conventional (you can see the same pattern [here](https://github.com/FormidableLabs/spectacle/pull/591/files#diff-16ef0e6c92055f5ab24dd4cd04a48742R77))